.github/workflows/spellchecker.yml

```diff
@@ -48,6 +48,9 @@ jobs:
         run: |
           set +e
           for f in ${{ steps.filter.outputs.plugins_files }}; do
+            if [[ "$f" =~ \.json$ ]] ; then
+              continue
+            fi
```

Comment:
I think it would be better to update the "filter" step to have only ".pm" files

Reply:
Ok. I've pushed an attempt.
And added quotes, according to [this recommendation](https://github.com/dorny/paths-filter?tab=readme-ov-file#notes):
> It's recommended to quote your path expressions with ' or ". Otherwise, you will get an error if it starts with *.